src/graph/fixtures.ts

```diff
@@ -0,0 +1,94 @@
+import {
+  Imageness,
+  SavedItemStatus,
+  Videoness,
+} from '../generated/graphql/types';
+import { ListItemObject, RestResponse } from './types';
+
+export const testSavedItemFragment = (id) => {
+  return {
+    id: id,
+    status: SavedItemStatus.Unread,
+    url: 'test.com',
+    isFavorite: true,
+    isArchived: true,
+    _createdAt: 1677818995,
```

Comment:
for future refactoring, can get other values as inputs or set default 

something like `_createdAt ?? defaultValue`